src/hpa_hooks.c

```diff
@@ -37,10 +37,27 @@ hpa_hooks_purge(void *ptr, size_t size) {
 	pages_purge_forced(ptr, size);
 }
 
-static void
-hpa_hooks_hugify(void *ptr, size_t size) {
+static bool
+hpa_hooks_hugify(void *ptr, size_t size, bool sync) {
+	/*
+	 * We mark memory range as huge independently on synchronous or
+	 * asynchronous hugification is used to have correct VmFlags set for
+	 * introspection and accounting purposes.  If synchronous hugification
```

Comment:
nit: I think you mean: "... independently on synchronous and asynchronous hugification to have correct VmFlags set ..."?

Reply:
Rewrote the sentence to better capture indent, I hope it is better now.